packages/eddsa-ticket-pcd/src/EdDSATicketPCD.ts

```diff
@@ -46,53 +52,85 @@ export enum TicketCategory {
  * when verifying tickets.
  */
 export interface ITicketData {
-  // the fields below are not signed and are used for display purposes
-
+  // The fields below are not signed and are used for display purposes.
   attendeeName: string;
   attendeeEmail: string;
   eventName: string;
   ticketName: string;
   checkerEmail: string | undefined;
-
-  // the fields below are signed using the server's private eddsa key
-
-  ticketId: string; // primary key uuid of the ticket's `devconnect_pretix_tickets` entry
-  eventId: string; // primary key uuid of the event's `pretix_events_config` entry
-  productId: string; // primary key uuid of the ticket's `devconnect_pretix_items_info` entry
+  // The fields below are signed using the server's private eddsa key.
+  ticketId: string; // Primary key uuid of the ticket's `devconnect_pretix_tickets` entry.
+  eventId: string; // Primary key uuid of the event's `pretix_events_config` entry.
+  productId: string; // Primary key uuid of the ticket's `devconnect_pretix_items_info` entry.
```

Comment:
worth specifying that these can be used by 3rd parties to represent tickets that don't necessarily have to live in the Zupass tables. more general descriptions make sense here. eg. ticketId is a unique identifier of the ticket, an event id uniquely identifies an event, and a product id uniquely identifies the type of ticket (eg. General Admission vs. Volunteer, etc.)